projects/the-application/src/assets/i18n/main/te.json

```diff
@@ -0,0 +1,18 @@
+{
+    "ADD_TO_HOME_SCREEN": "హోమ్ స్క్రీన్‌కు జోడించండి",
+    "CLOSE": "మూసివేయబడింది",
+    "DESCRIPTION": "మానవులు, కుక్కలు మరియు పిల్లుల కోసం మెమరీ కార్డ్ గేమ్",
+    "FULL_STOP": ".",
+    "GAME": "ఆట",
+    "HIGH_SCORES": "అధిక స్కోర్లు",
+    "INSTALL": "ఇన్‌స్టాల్ చేయండి",
+    "INSTALL_MEMORY_GAME": "మెమరీ గేమ్‌ను ఇన్‌స్టాల్ చేయండి",
+    "PLEASE_RELOAD": "దయచేసి మళ్లీ లోడ్ చేయండి",
+    "PLEASE_FINISH_OR_RESET_GAME": "దయచేసి ఆట పూర్తి చేయండి లేదా రీసెట్ చేయండి",
+    "RECENT_SCORES": "ఇటీవలి స్కోర్‌లు",
+    "REFRESH": "రిఫ్రెష్ చేయండి",
+    "SETTINGS": "సెట్టింగులు",
```

Comment:
Can you add TAP and THEN?

See https://github.com/tedaky/memory-game/pull/16/files#diff-47c31de70082a49e837b88ef47ba61dc365e3d92008c6d09cbacc1a0edbfb98a

Reply:
Added definition for TAP and its good for THEN as well. There is no contextual definition for both TAP and THEN. One definition is enough